### DISCUSSION TITLE
Pillow EOL Versions?

### DISCUSSION THREAD

**Discussion by loganbush789** [python-pillow/Pillow#5097] [Q&A] [answered]
I am trying to find out if there are any end-of-life versions of Pillow, and if so, when do these versions typically become EOL/unsupported? Thanks.

**Comment by nulano** [comment, marked as answer]
Does the following help?

From https://pillow.readthedocs.io/en/stable/releasenotes/index.html:

> Pillow is released quarterly on January 2nd, April 1st, July 1st and October 15th. Patch releases are created if the latest release contains severe bugs, or if security fixes are put together before a scheduled release.
>
> Please use the latest version of Pillow. Functionality and security fixes should not be expected to be backported to earlier versions.

**Comment by loganbush789** [comment]
Thank you!
